### PR TITLE
Added JSON file as config for converter

### LINKED ISSUE #2: Use JSON file for configuration of api and accounts

### PR DESCRIPTION
**Pull request by itay101**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "token": "XXX",
+  "budget_id": "XXXX",
+  "accounts": [
+    {
+      "account_id": "XXX",
+      "type": "from AccountTypes",
+      "file_path": "XXXXX",
```
Comment (ventorero) [inline config.json:8]: why do we need file_path and export_file_path ?

File: main.py [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
-        transactions = [*transactions, *file.get_transactions()]
-    response = requests.post(f"https://api.youneedabudget.com/v1/budgets/{BUDGET_ID}/transactions", headers={"Authorization": f"Bearer {TOKEN}"},
+    for account in accounts:
+        processor = AccountTypeToProcessor().get_processor_by_type(account["type"])
+        account_id = account["account_id"]
+        file_path = account["file_path"]
+        export_file_path = account["export_file_path"]
+        file_processor = processor(file_path=file_path, export_file_path=export_file_path, account_id=account_id)
+        transactions = [*transactions, *file_processor.get_transactions()]
+    response = requests.post(f"https://api.youneedabudget.com/v1/budgets/{budget_id}/transactions", headers={"Authorization": f"Bearer {token}"},
```
Comment (ventorero) [inline main.py]: maybe instead of `budget_id` on `consts`, we can implement the whole URL under `const` with `budget_id` ?

Comment (ventorero) [reply main.py]: WDYT ?

**Review by itay101 — COMMENTED** [review]
Fixed lint

**Review by ventorero — APPROVED** [review]

**Pull request merged**